tests/5.0/metadirective/test_metadirective_arch_is_nvidia.c

```diff
@@ -0,0 +1,58 @@
+//===---test_metadirective_is_nvidia.c -------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+// 
+// Test for metadirectives based on OpenMP 5.0 examples metadirective.1-3.c
+//
+////===---------------------------------------------------------------------===//
+
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 100
+
+int metadirective1() {
+   
+   int v1[N], v2[N], v3[N];
+
+   int which_device;
+   int errors = 0;
+
+   for(int i=0; i<N; i++) { 
+      v1[i] = (i+1); 
+      v2[i] = -(i+1); 
+   }
+
+   #pragma omp target map(to:v1,v2) map(from:v3) device(0)
```

Comment:
There is an implicit assumption here that is not supported by the specification. device(0) need not be an Nvidia device.   The test will return true for all devices. The only guarantee the spec provides is 

> When called on the host device, it will return the same value as the
22 omp_get_initial_device routine.

Reply:
@spophale new commit addresses this.